configure.ac

```diff
@@ -4706,9 +4706,13 @@ esac
 
 case "$TARGET" in
 *32* | ARM | X86 | POWERPC | MIPS | SPARC | WASM | ARM6432)
+    AC_DEFINE([MONO_HOST32], [1], [Target is 32bit])
```

Comment:
This isn't used but I like the idea instead of `#if SIZEOF_VOID_P == 4`.
Note that in the Windows world this is summed up as `#if _WIN64` or `#ifdef _WIN64`.

Reply:
if this is about the target, shouldn't the variable be called `MONO_TARGET32` and `MONO_TARGET64` then?

Reply:
It is supposed to be about the host but it looks like I messed it up.

Reply:
Both would be reasonable, if done correctly.